src/Tracker/index.js

```diff
@@ -10,26 +10,6 @@ let shouldSendEvents = false
 const client = window.location.hostname
 const sdk_version = process.env.SDK_VERSION
 
-const RavenTracker = Raven.config('https://6e3dc0335efc49889187ec90288a84fd@sentry.io/109946', {
-  environment: process.env.NODE_ENV,
-  release: sdk_version,
-  debug: true,
-  autoBreadcrumbs: {
-    console: false
-  },
-  breadcrumbCallback: (crumb) => {
-    const isOnfidoXhr = crumb.category === 'xhr' && isOnfidoHostname(crumb.data.url)
-
-    const isOnfidoClick = crumb.category === 'ui.click' && crumb.message.includes('.onfido-sdk-ui')
-
-    const shouldReturnCrumb = isOnfidoXhr || isOnfidoClick
-
-    return shouldReturnCrumb ? crumb : false
-  },
-  whitelistUrls: [/onfido[A-z.]*\.min.js/g],
```

Comment:
small note, in the view sentry version you also have this setting:

https://docs.sentry.io/error-reporting/configuration/?platform=browser#whitelist-urls

But i'm ok with only adding it in next story as chatted before

Reply:
I can bring it back but I don't think it's working

Reply:
I'll put it back just in case, then in the next story I can assess if it's work correctly or it needs to be removed